util/Minimization_Monitor/data_xtrct/ush/find_cycle.pl

```diff
@@ -148,7 +148,6 @@
          $hr_ctr = $hr_ctr - 1;
          
          $newdir = "${dirpath}/${sortmm[$ctr]}/${hrs[$hr_ctr]}/${lcm}";
```

Comment:
I must have failed to commit this change when I ported this to S4 and Jet.  To enable plotting data from the global workflow, there should be a check for `$newdir`.  If it doesn't exist, then set `$newdir` to `"${dirpath}/${sortmm[$ctr]}"`.

Reply:
OK, I've added a check for $newdir and set it to "${dirpath}/${sortmm[$ctr]}" if the default location doesn't exist.